utils/tangleid/TangleID.js

```diff
@@ -23,7 +23,6 @@ class TangleID {
     const namespace = extension.getNamespace();
     extension.setup(this.api);
```

Comment:
Why do we need to setup CoreAPI into extension?
The service provider should be different right?

Reply:
You may misunderstood my question.
We do need this commit to separate those providers by using extensions.

I mean since the new usage of tangleid api is `tangleid.api` and `tangleid.local`, the `extension.setup(this.api);` here is not what we need, because it register functions of CoreAPI into extension and we don't use `tangleid.local.api` to invoke functions.

Reply:
Yes, you are right, and I got your point. 
Since we don't need to set up the CoreAPI into extension, maybe we could remove this line and `setup` method in the `BaseExtension`. 🤔

Reply:
Yes. I think we can remove that method !

Reply:
After consolidating Core API, @ender503 should mention it in another Markdown-based documentation entry.